fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1184,6 +1184,10 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			ASSERT(!deletes.empty() || !a.batchToCommit->HasDeleteRange());
 			rocksdb::WriteOptions options;
 			options.sync = !SERVER_KNOBS->ROCKSDB_UNSAFE_AUTO_FSYNC;
+			if (SERVER_KNOBS->ROCKSDB_DISABLE_WAL_EXPERIMENTAL) {
+				options.disableWAL = true;
+				options.sync = false;
```

Comment:
What's the reason to set this false when WAL is disabled? Is it possible to test these two options separately?

Reply:
my bad.. That would affect the disk writes as well. So, removed it.

Reply:
It seems #8443 needs the same change, probably with a new PR. @neethuhaneesha